sidebars.js

```diff
@@ -84,6 +84,7 @@ module.exports = {
         'how-to/web-embeds',
         'how-to/windows-arm',
         'how-to/windows-taskbar',
+        'how-to/sandbox',
```

Comment:
Looks like this file doesn't exist anymore and is complaining now when deploying from `main`.